js-foundations.js

```diff
@@ -2,24 +2,60 @@
 // Coding practical questions
 
 // 1. Write a function that takes a number as an argument and decides if the number is evenly divisble by three or not. Use the test cases 15, 0, 385, -7. Expected output: "15 is divisible by three", "0 is divisible by three", "385 is not divisble by three", "-7 is not divisble by three".
-
-
+function divNum (num){
+    if(num%3 === 0){
+        return num + " is divisble by three"
+    }else if(num%3 === 1){
+        return num + " is not divisble by three"
+    }else if(num%3 !==0){
+       return num + " is not divisble by three"
+    }
+}
+console.log(divNum(15));
+console.log(divNum(0));
+console.log(divNum(385));
+console.log(divNum(-7));
 
```

Comment:
This totally works, and that is the goal of code, but you can shorten this up a bit since we know there are only two possible outcomes. Just a thought.